ggshield/cmd/auth/login.py

```diff
@@ -39,7 +39,7 @@ def validate_login_path(
     if instance is None:
         return sso_instance, sso_login_path
 
-    config_parsed_url = clean_url(config.dashboard_url)
+    config_parsed_url = clean_url(instance)
```

Comment:
The `config` parameter becomes unused, should we remove it entirely from the function's signature?